### DISCUSSION TITLE
Where is the value in the level of tags in the alarm rule configured?[Feature]

### DISCUSSION THREAD

**Discussion by 2han9b1n** [apache/skywalking#12106] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Where is the value in the level of tags in the alarm rule configured?
Besides ‘WARNING’ there are other？
![1713170371566](https://github.com/apache/skywalking/assets/46619968/bd6daa92-0dfd-4d6a-8a0d-d67757051b97)

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
They are tags, they could be anything.
